specs/agents/otel-distribution.md

```diff
@@ -54,8 +54,9 @@ collector is used then the user might opt in to also collect those.
 
 These metrics are not captured by default in order to prevent duplicated metrics when they are also captured by a collector.
 
+Configuration to enable: `ELASTIC_OTEL_SYSTEM_METRICS_ENABLED`
```

Comment:
Should we call them "system metrics" or "host metrics"?
I know the section title in this doc has used "System metrics", but if we are talking about enabling/disabling the equivalent of the [OTel Collector hostmetricsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md), then `ELASTIC_OTEL_HOST_METRICS_ENABLED` migth be a better name.

The OTel JS project calls it "host metrics" as well: https://www.npmjs.com/package/@opentelemetry/host-metrics

Reply:
I think it's named after the specification and the namespace https://opentelemetry.io/docs/specs/semconv/system/system-metrics/
They refer to _host_ with a different meaning: "Resource attributes related to a host, SHOULD be reported under the host.* namespace."